sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/ReflectionUtils.java

```diff
@@ -198,6 +198,10 @@ public static ConnectionPolicy getConnectionPolicy(CosmosClientBuilder cosmosCli
         return get(ConnectionPolicy.class, cosmosClientBuilder, "connectionPolicy");
     }
 
+    public static ConnectionPolicy getConnectionPolicy(CosmosAsyncClient cosmosAsyncClient) {
```

Comment:
why not `BridgeInternal.getContextClient(client).getConnectionPolicy()`

Reply:
Fixed in next iteration.